ci/builders/README.md

```diff
@@ -374,9 +374,126 @@ be relative to the checkout directory.
 
 ### Global Tests
 
-Global tests follow the same format as build tests but defined at the top
-level. The main difference from local tests is that global tests have access
-to the sub-build artifacts.
+Tests in this section run on a separate bot as an independent sub-builds.
+As opposed to tests running within builds, global tests have access to the
+the outputs of all the builds running in the same orchestrator build. A
+use case for global tests is to run flutter/framework tests using the
+artifacts generated by an specific engine build.
+
+Global tests currently support two different scenarios:
+
+* flutter/flutter tests with [tester](https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2/tester.py)
+  recipe. This workflow checks out flutter/flutter to run any of the existing
```

Comment:
Ah okay this is really valuable information. So tester.py will automatically check out flutter/flutter to run tests. Cool!

Reply:
ack